tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -544,6 +544,21 @@ ARDUINO_PROJECT_TARGETS := $(foreach TARGET,$(ALL_PROJECT_TARGETS),$(if $(findst
 generate_arduino_zip: $(ARDUINO_PROJECT_TARGETS) $(ARDUINO_LIBRARY_ZIPS)
 	python tensorflow/lite/micro/tools/make/merge_arduino_zips.py $(PRJDIR)/tensorflow_lite.zip $(ARDUINO_LIBRARY_ZIPS)
 
+# API for external project generation scripts to retrieve lists of files needed
+# for particular platforms. For more details see:
+# https://docs.google.com/document/d/1gfbpBtiivYMenCDIvJb-m5RIV4rKAR5dvdAJC5LLv8o
```

Comment:
can we move the doc to markdown (maybe `tensorflow/lite/micro/docs/external_integration.md`?).

Might be worth doing that as part of the first PR so that we can avoid stale links in the code?